source/includes/fact-viewing-favorite-queries.rst

```diff
@@ -12,4 +12,22 @@
 
    .. step:: Click the :guilabel:`Favorites` button in the past queries pane.
 
-      Select favorite queries.
+      Select your favorited syntax.
+
+      - If your favorited syntax is a query, only a :guilabel:`Filter` 
+        statement displays in the :guilabel:`Favorites` list view 
+        along with copy :icon-lg:`Copy` and delete :icon-lg:`Trash` 
+        icons.
+
+      - If your favorited syntax is a bulk update statement, a 
+        :guilabel:`Filter` and :guilabel:`Update` statement display in 
+        the :guilabel:`Favorites` list view along with copy 
+        :icon-lg:`Copy`, delete :icon-lg:`Trash`, and 
+        :icon-lg:`OpenNewTab` open in modal icons.
+
+        .. note::
+            
+            Clicking the open modal icon, opens the bulk update statement 
```

Comment:
While this is true, I also think it it's worth noting above that clicking on a regular query will populate the filter bar and simply clicking on the update will open the model (you don't have to click an icon)